api/tests/core/opening_hours/test_api.py

```diff
@@ -28,9 +29,10 @@ def validate_timespans(fetched_timespans, expected_timespans):
     fetched_timespans example: [NumericRange(600, 720), NumericRange(780, 1200)]
     expected_timespans example: [["10:00", "12:00"], ["13:00", "20:00"]]
     """
-    fetched_timespans = sorted(fetched_timespans, key=lambda ts: ts.lower)
     expected_timespans = sorted(expected_timespans, key=lambda oh: oh[0])
-
+    fetched_timespans = sorted(fetched_timespans, key=lambda ts: ts.lower)
+    # remove unused argument and canonize string représentation
```

Comment:
```suggestion
    # remove unused argument and canonize string representation
```
hon hon hon baguette